Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM nimlang/nim:1.0.0-ubuntu as build
+
+ARG NITTER_HOSTNAME
+
+ENV NITTER_HOSTNAME nitter.net
+
+WORKDIR /build
+
+COPY . /build
+
+RUN apt-get update && apt-get install -y libsass-dev
+
+RUN nimble build -d:release -d:hostname="$NITTER_HOSTNAME" --accept
```

Comment:
actually i'm not really sure if that replacement of `$NITTER_HOSTNAME` is working correctly